types/uatp.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+var Type = require('../type')
+
+module.exports = Type({
+  name: 'UATP',
+  digits: 15,
+  pattern: /^1\d{14}$/,
+  eagerPattern: /^1/,
+  groupPattern: /(1\d{3})(\d{1,5})?(\d{1,6})?/
```

Comment:
Google search indicates this is not right:

https://www.google.com/search?q=uatp+card&source=lnms&tbm=isch&sa=X&ved=0ahUKEwiQr67Y6oPZAhUO1mMKHfGMAY0Q_AUICigB&biw=2127&bih=1321#imgrc=nY11YBAnx6I-aM: